output/components.sasslint.txt

```diff
@@ -3,11 +3,14 @@
 > sass-lint -v -q
 
 
+src/ObjectViewer/JSONLike/JSONLike.scss
+  65:21  warning  Hex notation should all be upper case  hex-notation
```

Comment:
prettier put it in lowercase ... what should we do ?

Reply:
desactivate prettier for scss or change the lint rule.